lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueNotificationRepository.java

```diff
@@ -80,10 +78,14 @@ public LightblueNotificationRepository(LightblueClient lightblue, String[] entit
             DataBulkRequest updateEntities = new DataBulkRequest();
```

Comment:
Cool, I made the issue to look at all uses of bulk updates, but feel free to comment there if you find more. I'll copy this one. Ty